src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/data/DocumentCategory.kt

```diff
@@ -0,0 +1,5 @@
+package uk.gov.justice.digital.hmpps.hmppsresettlementpassportapi.data
+
+enum class DocumentCategory(val displayName: String) {
+  EMPLOYMENT_SKILLS_WORK("Employment, skills and work"),
```

Comment:
The first thing we want to support is licence conditions, what's with this category?

Reply:
I got this from the prototype. this is now changed!